demo/sections/DesignTokens.vue

```diff
@@ -0,0 +1,400 @@
+          </p-code>, <p-code inline>
+            input
+          </p-code>, <p-code inline>
+            tag
+          </p-code>, etc.
+        </li>
+        <li>
+          <p-code inline>
+            {variant}
+          </p-code> is for alternate variations of the token, e.g. <p-code inline>
+            default
+          </p-code>, <p-code inline>
+            primary
+          </p-code>, <p-code inline>
+            danger
+          </p-code>, etc.
+        </li>
+        <li>
+          <p-code inline>
+            {property}
+          </p-code> is the CSS property the token is intended for, e.g. <p-code inline>
+            bg
+          </p-code>, <p-code inline>
+            border
+          </p-code>, <p-code inline>
+            text
+          </p-code>, etc.
+        </li>
+        <li>
+          <p-code inline>
+            {state}
+          </p-code> is the state of the component, e.g. <p-code inline>
+            hover
+          </p-code>, <p-code inline>
+            active
+          </p-code>, <p-code inline>
+            focus
+          </p-code>, etc.
+        </li>
+      </ul>
+      <p class="design_tokens__paragraph">
+        Examples:
+      </p>
+      <p-code class="design_tokens__code-block">
+        --p-color-input-bg
+        <br>
+        --p-color-input-border
+        <br>
+        --p-color-input-text
+        <br>
+        --p-color-input-placeholder
+        <br>
+        // Format: --p-{category}-{pattern}-{property}
+      </p-code>
+      <p-code class="design_tokens__code-block">
+        --p-color-button-primary-bg-hover
+        <br>
+        --p-color-button-danger-bg-active
+        <br>
+        --p-color-button-default-border-hover
+        <br>
+        // Format: --p-{category}-{pattern}-{variant}-{property}-{state}
+      </p-code>
+    </template>
+
+    <template #backgrounds-and-borders>
+      <p class="design_tokens__paragraph">
+        There are 3 base levels of background colors. If we encounter a design calling for more levels of background colors than this, we should reconsider the design.
+      </p>
+      <div class="border border-default rounded-md my-4 p-6">
+        <p><strong>Level 0:</strong> The base background color of the application, applied globally.</p>
+        <div class="p-background mt-4 p-6 rounded-md">
+          <p>
+            <strong>Level 1:</strong> Add the class <p-code inline>
+              p-background
+            </p-code> directly to a container to apply the level 1 background color. Used on cards by default.
+          </p>
+          <div class="p-background mt-4 p-6 rounded-md">
+            <p>
+              <strong>Level 2:</strong> Adding the class <p-code inline>
+                p-background
+              </p-code> directly to a container will automatically apply the level 2 background color when it's nested within a level 1 background, using a simple <p-code inline>
+                .p-background + .p-background {}
```

Comment:
I don't think this is correct. It should just be `.p-background .p-background` right?